content/en/history/bibliography/_index.md

```diff
@@ -2,10 +2,6 @@
 title: Bibliography
 weight: 5
 type: docs
-aliases:
- - /bibliography/
```

Comment:
Does removing this break the bibliography "shortcut" url:
https://interlisp.org/bibliography

Reply:
Yes it would - where is that shortcut used?  I thought these pages besides not being indexed weren't ever referenced by anything Google knows about.  Maybe I misunderstood.

I would like to start pruning out old unused aliases.  But, on the other hand, I also don't want to be overzealous and break links.

Reply:
That bibliography URL (shortcut) was referenced in Andrew's paper/presentation of several months ago. So anyone who finds Andrew's presentation on-line will see [https://interlisp.org/bibliography](https://interlisp.org/bibliography) as the URL to our Bibliography.

Reply:
Add alias back in